lib/config.ts

```diff
@@ -75,7 +76,10 @@ export const config: Linter.Config[] = [
 				'node_modules/@types',
 			],
 			'import-x/resolver': {
-				node: allExtensions,
+				node: {
+					extensions: allExtensions,
+					caseSensitive: !isFileSystemCaseInsensitive(),
```

Comment:
I can't find this option documented anywhere...can you link the docs for this or the code where its actually used?